src/components/settings/Settings.h

```diff
@@ -95,9 +95,13 @@ namespace Pinetime {
       uint32_t GetStepsGoal() const { return settings.stepsGoal; };
 
     private:
-      Pinetime::Drivers::SpiNorFlash& spiNorFlash;
+      Pinetime::Controllers::FS& fs;
+
+      static constexpr uint32_t settingsVersion = 0x0001;
       struct SettingsData {
 
+        uint32_t version = settingsVersion;
```

Comment:
I'm not sure this is needed, by we could save a few bytes here by ordering the fields of the struct by size (uint32_t first, then uint16_t and then uint8_t). This will reduce the number of padding bytes added by the compiler.
Also, all the enums used in this struct could be constrained to let's say uint8_t to ensure that the compiler won't use a larger storage for the enum, and won't implicitely change it from one version to another.

Reply:
Thank you, I will review the recommendations.